code/SSResqueLogger.php

```diff
@@ -36,42 +15,17 @@ public function __construct($verbose = false) {
 	 * @return null
 	 */
 	public function log($level, $message, array $context = array()) {
-		$fh = fopen($this->path, 'a');
-		if($this->verbose) {
-			fwrite($fh, '[' . $level . '] [' . strftime('%T %Y-%m-%d') . '] ' . $this->interpolate($message, $context) . PHP_EOL);
-			return;
-		}
+		parent::log($level, $message, $context);
 
-		if(!($level === Psr\Log\LogLevel::INFO || $level === Psr\Log\LogLevel::DEBUG)) {
-			fwrite($fh, '[' . $level . '] ' . $this->interpolate($message, $context) . PHP_EOL);
-		}
-		fclose($fh);
+		// log this with SS_Log so that SS_Log writers will act on the error if configured
+		SS_Log::log($this->interpolate($message, $context), $this->convertLevel($level));
```

Comment:
you could pass $context['stack'] (if exists) which is an Exception to SS_Log and I think the email subject formatting might be nicer? See worker.php line 248